tests/integration/poster/poster_server_mock.py

```diff
@@ -0,0 +1,63 @@
+from http.server import BaseHTTPRequestHandler, HTTPServer
+import re
+import socket
+import requests
+from threading import Thread
+from http.server import  HTTPServer
+import json
+
+
+class PosterServerMock(BaseHTTPRequestHandler):
+    """Generic Mock for Poster interactions"""
+
+    PATTERN = None
+    port = None
+
+    def start(self):
+        self.port = self.free_port(self)
+        server = HTTPServer(("localhost", self.port), self)
+        thread = Thread(target=server.serve_forever)
+        thread.setDaemon(True)
+        thread.start()
+
+    def free_port(self):
+        """Returns free port
+
+        :return: Port
+        """
+        sock = socket.socket(socket.AF_INET, type=socket.SOCK_STREAM)
+        sock.bind(("localhost", 0))
+        address, port = sock.getsockname()
+        sock.close()
+        return port
+
+    def do_GET(self):
+        if re.search(self.PATTERN, self.path):
+            self.send_response(requests.codes.ok)
```

Comment:
@paulodamaso It's ok, but I'd like to use here standart lib from python
`from http import HTTPStatus`

`self.send_response(HTTPStatus.OK)`